packages/codebytes/src/editor.tsx

```diff
@@ -40,30 +41,28 @@ type EditorProps = {
   language: LanguageOption;
   text: string;
   onChange: (text: string) => void;
-  onCopy?: (text: string, language: string) => void;
   snippetsBaseUrl?: string;
+  onCopy?: CodebytesChangeHandler;
 };
 
 export const Editor: React.FC<EditorProps> = ({
   language,
   text,
   hideCopyButton,
   onChange,
-  onCopy,
   snippetsBaseUrl,
+  onCopy,
 }) => {
   const [output, setOutput] = useState('');
   const [status, setStatus] = useState<'ready' | 'waiting' | 'error'>('ready');
   const [isCodeByteCopied, setIsCodeByteCopied] = useState(false);
   const onCopyClick = () => {
     if (!isCodeByteCopied) {
-      navigator.clipboard
-        .writeText(text)
-        // eslint-disable-next-line no-console
-        .catch(() => console.error('Failed to copy'));
-      onCopy?.(text, language);
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      navigator.clipboard.writeText(text);
```

Comment:
why are we removing the catch here?

Reply:
removed it, because i didn't think it was adding value, just logging an error to the console for the developer. can add it back if we really wish.

Reply:
the log isnt huge but we probably want to catch an error so it doesnt throw and break the page

Reply:
absolutely ok, can add back 👍